docs(vpc): Troubleshooting debian bullseye no default route int-add-vpc (#2248)
scaleway/docs-content: network/vpc/troubleshooting/private-dns-dhcp-not-working.mdx

In this case, if you are using `systemd-networkd` and do not have a route to our service IP address `169.254.169.254`, you should try updating and upgrading the packages on your system.

> There is also a known [bug](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=867625) affecting **Debian Bullseye** that prevents DHCP Client to apply correctly the default route. For this specific bug, we recommend to use another distribution like **Debian Bookworm**.

Reviewer: ```suggestion
There is also a known [bug](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=867625) affecting **Debian Bullseye** that prevents DHCP clients  from correctly applying the default route. For this specific bug, we recommend using another distribution, such as **Debian Bookworm**.
```

> There is also a known [bug](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=867625) affecting **Debian Bullseye**, that prevents DHCP clients  from correctly applying the default route. For this specific bug, we recommend using another distribution, such as **Debian Bookworm**.

Reviewer: ```suggestion
There is also a known [bug](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=867625) affecting **Debian Bullseye**, that prevents DHCP clients from correctly applying the default route. For this specific bug, we recommend using another distribution, such as **Debian Bookworm**.
```

Reviewer: Sorry, just noticed the double space!
